### ISSUE TITLE
Nesting Storage boxes

### ISSUE THREAD

**Issue by DasWookie** [kigster/laser-cutter#3]
Any chance of creating an option for folding boxes, which could be lasered out and then folded, similar to something like these storage boxes:
http://www.thingiverse.com/thing:673699

In an ideal world being able to specify sizes ranging from 3" square up to possibly as large as 9x9x4".

**Comment by kigster** [comment]
@DasWookie what do you mean by "folded"?  

I can see how doing a cut-out so that the box is more like an open drawer be a useful feature for many (myself included!).  I can see what it would take.

**Comment by DasWookie** [comment]
Folded == going from the flat lasered out piece and then folding up the edges and such into the final form of the box. I have been using the example URL project files for the meanwhile, but without having something parametric where one can set arbitrary box sizes it limits the stackability and small parts storage solution I really desire. My hope was to be able to get something where I could generate templates just like the Thingiverse project, via laser-cutter, but for the sizes of my various electronic components and other small parts, as necessary. My current "storage" solution of envelopes all crammed into 3 old cardboard boxes from printer paper is... well... utterly and completely inefficient! LOL I tend to spend more time searching for parts I know I have, but can't find... order more... and 3 days (or more) after the new order arrives I finally stumble upon the envelope of the parts I already had.

**Comment by kigster** [comment]
This is exactly what this library does. Have you tried creating your laser-cutting template on http://makeabox.io/ ?  This is my site that uses this library to generate boxes that, as you call it, fold.

**Comment by DasWookie** [comment]
Yes, the problem is I need an opening on the front and top I can get my big paws in, with the beveled opening... like the thingiverse project does:
http://thingiverse-production-new.s3.amazonaws.com/renders/12/7a/05/23/56/Organizer_bins_preview_featured.jpg

**Comment by DasWookie** [comment]
The reason why I'd initially contacted you / written up the "issue" was your library seems to be the closest to supporting the functionality for getting that opening / cutout in a fully parametric box. Love the library, it just SLIGHTLY misses on my use case I was trying to find... it's damn impressive otherwise. :)
